rest/src/main/scala/io/udash/rest/raw/RestRequest.scala

```diff
@@ -2,9 +2,9 @@ package io.udash
 package rest
 package raw
 
-import com.avsystem.commons.meta._
+import com.avsystem.commons.meta.*
 import com.avsystem.commons.misc.{AbstractValueEnum, AbstractValueEnumCompanion, EnumCtx}
-import com.avsystem.commons.rpc._
+import com.avsystem.commons.rpc.*
```

Comment:
nit: in the future let's try to avoid reformatting unchanged files, the diff is huge anyway